mifosng-android/src/main/res/values/strings.xml

```diff
@@ -199,6 +199,8 @@
     <string name="add_payment_detail">Add Payment Detail</string>
     <string name="generate_new">Generate New</string>
     <string name="currency">Currency</string>
+    <string name="Channel_Name">Channel_Name</string>
+    <string name="Channel_Description">Channel_Description</string>
```

Comment:
Where are these 2 strings being used in your code?

Reply:
These are being used in the method- createNotificationChannel() in class SyncSurveyDialogFragment Line number 106 and 107.